### PR TITLE
Additional light client metrics 

### PR DESCRIPTION
**Pull request by eserilev**
## Issue Addressed

Add some additional light client metrics plus one small bugfix to the get light client updates endpoint

### CODE REVIEW

**Review by dapplion — APPROVED** [review]
LGTM!

File: beacon_node/beacon_chain/src/metrics.rs [hunk 1]
```diff
@@ -1937,6 +1937,22 @@ pub static LIGHT_CLIENT_SERVER_CACHE_PREV_BLOCK_CACHE_MISS: LazyLock<Result<IntC
         )
     });
 
+pub static LIGHT_CLIENT_SERVER_CACHE_PROCESSING_REQUESTS: LazyLock<Result<IntCounter>> =
+    LazyLock::new(|| {
+        try_create_int_counter(
+            "light_client_server_cache_processing_requests",
```
Comment (jimmygchen) [inline beacon_node/beacon_chain/src/metrics.rs]: Small nit to prefix the metric with `beacon_` for consistency with other beacon chain metrics and to make it easier to distinguish BN metrics in Grafana queries:

```suggestion
            "beacon_light_client_server_cache_processing_requests",
```

File: beacon_node/beacon_chain/src/metrics.rs [hunk 2]
```diff
@@ -1937,6 +1937,22 @@ pub static LIGHT_CLIENT_SERVER_CACHE_PREV_BLOCK_CACHE_MISS: LazyLock<Result<IntC
...
+            "Count of all requests to recompute and cache updates",
+        )
+    });
+
+pub static LIGHT_CLIENT_SERVER_CACHE_PROCESSING_SUCCESSES: LazyLock<Result<IntCounter>> =
+    LazyLock::new(|| {
+        try_create_int_counter(
+            "light_client_server_cache_processing_successes",
```
Comment (jimmygchen) [inline beacon_node/beacon_chain/src/metrics.rs]:
```suggestion
            "beacon_light_client_server_cache_processing_successes",
```

**Review by jimmygchen — CHANGES_REQUESTED** [review]
Thanks, the change looks good to me! Just a small nit on the metric name.

**Review by jimmygchen — APPROVED** [review]
Looks great, thanks!

**Comment by jimmygchen** [comment]
@mergify queue

**Comment by michaelsproul** [comment]
@mergify requeue

**Pull request merged**